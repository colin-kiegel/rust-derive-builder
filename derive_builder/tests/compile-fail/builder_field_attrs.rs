#[macro_use]
extern crate derive_builder;

#[derive(Debug, PartialEq, Default, Builder, Clone)]
pub struct Lorem {
    ok: String,

    // This foolish repr() attribute generates an unused attribute warning
    #[builder_field_attrs(
        #[no_such_attribute]
    )]
    broken: String,
}

fn main() {}
