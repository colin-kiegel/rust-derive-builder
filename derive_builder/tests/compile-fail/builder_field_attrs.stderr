error: cannot find attribute `no_such_attribute` in this scope
 --> tests/compile-fail/builder_field_attrs.rs:9:27
  |
9 |     #[builder_field_attrs(no_such_attribute)]
  |                           ^^^^^^^^^^^^^^^^^
